Formula/d/dive.rb

```diff
@@ -18,7 +20,16 @@ class Dive < Formula
   depends_on "go" => :build
 
   def install
-    system "go", "build", *std_go_args(ldflags: "-s -w -X main.version=#{version}")
+    if build.head?
+      cd buildpath do
+        commit_hash = `git rev-parse --short HEAD`.strip
+        version_string = "HEAD@#{commit_hash}"
+        system "go", "mod", "download"
+        system "go", "build", *std_go_args(ldflags: "-s -w -X main.version=#{version_string}")
+      end
+    else
+      system "go", "build", *std_go_args(ldflags: "-s -w -X main.version=#{version}")
+    end
```

Comment:
there is `Utils.git_head`

so it can be just `version = build.head? ? Utils.head : version`